event_operating_unit/security/event_operating_security.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="0">
+    <record id="ir_rule_event_allowed_operating_units"
+            model="ir.rule">
+        <field name="model_id" ref="event.model_event_event"/>
+        <field name="domain_force">['|', ('operating_unit_id','=', False),('operating_unit_id','in',[g.id for g in user.operating_unit_ids])]</field>
```

Comment:
You can use `user.operating_unit_ids.ids` instead of `[g.id for g in user.operating_unit_ids]`.